user-operator/src/test/java/io/strimzi/operator/user/operator/SimpleAclOperatorTest.java

```diff
@@ -113,14 +114,15 @@ public void testInternalCreate(TestContext context)  {
             List<scala.collection.immutable.Set<Acl>> capturedAcls = aclCaptor.getAllValues();
             List<Resource> capturedResource = resourceCaptor.getAllValues();
 
-            context.assertEquals(2, capturedAcls.size());
-            context.assertEquals(2, capturedResource.size());
+            context.assertEquals(2, capturedAcls.get(0).size());
+            context.assertEquals(1, capturedResource.size());
 
             context.assertEquals(res1, capturedResource.get(0));
-            context.assertEquals(res1, capturedResource.get(1));
+            //context.assertEquals(res1, capturedResource.get(1));
```

Comment:
If thi si snot needed, you can delete it. But I think it would make sense to add some more rules in this test to make sure the grouping by resource works properly. I.e. add at least one additional rule which will be using different resource.